src/components/Bookstate.jsx

```diff
@@ -41,13 +41,29 @@ const Books = () => {
     <>
       <ul className="booksUl">
         {bookData.map((book) => (
-          <li className="book" key={book.key}>
-            <p className="category">{book.category}</p>
-            <h2 className="title">{book.title}</h2>
-            <h3 className="author">{book.author}</h3>
-            <button type="button" onClick={() => handleRemoveBook(book)}>
-              Remove
-            </button>
+          <li className="bookCont" key={book.key}>
+            <div className="book">
+              <p className="category">{book.category}</p>
+              <h2 className="title">{book.title}</h2>
+              <h3 className="author">{book.author}</h3>
+              <button className="bookButton" type="button">Comments</button>
+              <button className="bookButton" type="button" onClick={() => handleRemoveBook(book)}>
+                Remove
+              </button>
+              <button className="bookButton" type="button">Edit</button>
```

Comment:
- [ ] [OPTIONAL] Hi, I really like your design and how you implemented the layout and colors. You did a great job of following the template. However, I noticed that your design is missing the vertical lines that separate the sections in the template. These lines are important to create a clear visual distinction between the buttons and to enhance the overall aesthetics. To add these lines, you can use the CSS border property on the elements that contain each section. For example:

```css
..bookButton {
  border-right: 1px solid #ccc; /* add a 1px solid gray border to the right of each section */
}

..bookButton:last-child {
  border-right: none; /* remove the border from the last button */
}
```

Please try to add these lines and resubmit your design for review. If you have any questions or need further assistance, feel free to reach out to me. Keep up the good work! 😊


<img width="661" alt="image" src="https://github.com/VelzckC0D3/bookstore-react-redux/assets/52934128/c03a77ca-816b-4517-b085-4c6b113c3b42">